src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,13 @@
+<hibernate-configuration>
+    <session-factory>

```

Comment:
Add dialect property name="hibernate.dialect"

Reply:
WARN: HHH90000025: HSQLDialect does not need to be specified explicitly using 'hibernate.dialect' (remove the property setting and it will be selected by default)

I have this warning whenever I add the dialect property explicitly. I've added it, but is it necessary? What does it improve exactly?

Reply:
Also part of my code refuses to work when add this property for some reason

Reply:
@AlexRosenkreuz you shouldn't use `HSQLDialect` here

you have a MySQL connector - therefore you should use `MySQL5Dialect` or `MySQL8Dialect` etc.

dialect helps Hibernate to construct SQL queries considering the underlying database, since each database has some specific syntax, more or less

Reply:
Oh, my mistake. Used ridiculously wrong dialect---